config/i18n.json

```diff
@@ -583,6 +583,25 @@
     "MissingClassDescription": "The build you're trying to view is for a character class you don't have.",
     "NewEmptyLoadout": "Create Empty Loadout",
     "NoBuildsFound": "No builds found. Try expanding your filtering criteria.",
+    "NoBuildsFoundExplainer": {
+      "Header": "No builds found. Here are possible reasons DIM couldn't find any sets:",
+      "AlwaysInvalidMods": "These mods don't fit into any of your items:",
```

Comment:
```suggestion
      "AlwaysInvalidMods": "These mods don't fit into any of the allowed items:",
```

Reply:
This change would be inaccurate, this actually points out mods the user can't fit into any of the items for this class, globally.

Reply:
maybe there is some way to make that a little more explicit/emphatic then? "any owned items"?